baselines/NER/docile_train_NER_multilabel.py

```diff
@@ -1004,10 +1032,11 @@ def compute_metrics(eval_pred):
     metrics = trainer.evaluate()
     trainer.log_metrics("eval", metrics)
 
-    # compute stats on train_sataset
-    trainer.eval_dataset = train_dataset
-    metrics = trainer.evaluate()
-    trainer.log_metrics("train-eval", metrics)
+    # compute stats on train_dataset (skip for synthetic dataset since it can be memory issues)
```

Comment:
"since it can introduce memory issues" or "since it can result in memory issues" ?